tidb/src/main/java/io/tidb/bigdata/tidb/JdbcConnectionProviderFactory.java

```diff
@@ -28,6 +28,8 @@ public class JdbcConnectionProviderFactory {
 
   public static JdbcConnectionProvider createJdbcConnectionProvider(ClientConfig config) {
     try {
+      // load JDBC drivers and register to DriverManager
+      Class.forName(ClientConfig.MYSQL_DRIVER_NAME);
       Class<?> provideClass = Class.forName(config.getJdbcConnectionProviderImpl());
```

Comment:
can you introduce what's the purpose of this forname

Reply:
You can find the purpose at this [link](https://dev.mysql.com/doc/connector-j/5.1/en/connector-j-usagenotes-connect-drivermanager.html).